template.yml

```diff
@@ -85,6 +85,22 @@ Parameters:
       - "true"
       - "false"
 
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: stax-orchestrator
+    Description: >-
+      Stax Workloads Orchestrator -
+      Interact with Stax to CUD (create-update-delete)
+      workloads in your AWS Account(s).
+    Author: Versent
+    SpdxLicenseId: Apache-2.0
+    LicenseUrl: LICENSE
+    ReadmeUrl: README.md
+    Labels: ['AWS', 'Stax', 'Serverless', 'Lambda', 'StepFunction']
+    HomePageUrl: https://github.com/Versent/stax-orchestrator.git
+    SemanticVersion: 0.1.6
```

Comment:
```suggestion
```
Let's remove this and allow it to be set by the build argument (CLI)

Reply:
Removed 👍